src/App/App.tsx

```diff
@@ -69,7 +69,7 @@ const App: FC<RouteComponentProps> = () => {
     const [dataLoaded, setDataLoaded] = useState(false)
 
     if (process.env.NODE_ENV !== 'production' && !process.env.JEST_WORKER_ID) {
-        axe()
+        // axe()
```

Comment:
@Aidenbuis Did you disable it because of some errors? If you set the version to "4.4.1" it should be resolved

Reply:
No it made the development environment really laggy, so I disabled it. Haven't looked into why that happens